cmd/environments/environments.go

```diff
@@ -25,6 +25,25 @@ func NewEnvironmentsCmd(
 				analytics.CmdRunEventProperties(cmd, "environments"),
 			)
 		},
+		PersistentPostRun: func(cmd *cobra.Command, args []string) {
+			outcome := analytics.SUCCESS
+
+			if _, hasErr := cmd.Annotations["err"]; hasErr {
+				outcome = analytics.ERROR
+			}
+			if cmd.HasHelpSubCommands() {
+				outcome = analytics.HELP
+			}
+
+			analyticsTracker.SendEvent(
+				viper.GetString(cliflags.AccessTokenFlag),
+				viper.GetString(cliflags.BaseURIFlag),
+				"CLI Command Completed",
+				map[string]interface{}{
+					"outcome": outcome,
+				},
+			)
```

Comment:
I'll be abstracting this out into a function. But while I have the first one written, do you want to check it @dbolson